test/e2e/fixtures.go

```diff
@@ -300,10 +304,47 @@ func setupTestForFlowAggregator(tb testing.TB, o flowVisibilityTestOptions) (*Te
 	if err := testData.deployFlowAggregator(ipfixCollectorAddr, o); err != nil {
 		return testData, v4Enabled, v6Enabled, err
 	}
-
+	err = testData.RestartAntreaAgentPods(defaultTimeout)
```

Comment:
That seems like a bit of a waste if `deployFlowAggregator` was a no-op. Does it increase test time?

Reply:
Yes, it does increase test time a little bit. I've removed it as it is not needed.